cloud_functions/climateiq_merge_output_cf/main.py

```diff
@@ -1,7 +1,98 @@
-from cloudevents import http
+import collections
+import csv
+import re
+
+import flask
+from google.cloud import storage
 import functions_framework
 
 
-@functions_framework.cloud_event
-def merge_scenario_predictions(cloud_event: http.CloudEvent) -> None:
-    return
+# Bucket name, where prediction outputs are stored.
+BUCKET_NAME = "climateiq-predictions"
+# File name pattern for the CSVs for each scenario and chunk.
+CHUNK_FILE_NAME_PATTERN = (
+    r"(?P<run_id>\w+)/(?P<prediction_type>\w+)/(?P<model_id>\w+)/"
+    r"(?P<study_area_name>\w+)/(?P<scenario_id>\w+)/(?P<chunk_id>\w+)\.csv"
+)
+# Directory in bucket to write merged files.
+OUTPUT_DIR = "merged"
+
+
+# This only handles merging flood data. Heat data will be divided into completely
+# different chunks.
+@functions_framework.http
```

Comment:
setting this up to trigger via HTTP is fine for now but ideally we'd chain all the steps together to run automatically after the initial manual request to begin the export pipeline

the tricky part is figuring out when all the spatialization cloud functions have completed for every chunk